static/bidder-params/connectad.json

```diff
@@ -0,0 +1,23 @@
+
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "ConnectAd S2S dapter Params",
+    "description": "A schema which validates params accepted by the ConnectAd Adapter",
+  
+    "type": "object",
+    "properties": {
+      "networkId": {
+        "type": "integer",
+        "description": "NetworkId"
+      },
+      "siteId": {
+        "type": "integer",
+        "description": "SiteId"
+      },
+      "bidfloor": {
```

Comment:
I noticed the doc PR has this listed as `floorprice`.

Reply:
oh boy, so true! we are accepting both values in the backend. I will update the docs accordingly